File: javascript/selectionSort.js
let my_array = [64, 34, 55, 22, 12, 45, 99]

let n = my_array.length

for(i = 0; i < n - 1; i ++){

    let min_index = 1

    for(j = i + 1; j <  n; j++){

        if(my_array[j] < my_array[min_index]){

            min_index = j
        }
    }
    let min_value = my_array.splice(min_index,1)

    my_array.splice(i, 0,Number(min_value[0]))
   
}

console.log("Sort array: ", my_array);

File: python/selectionSort.py
my_array = [64, 34, 55, 22, 12, 45, 99]

n = len(my_array)

for i in range(n - 1):

    min_index = 1
    print("i = ", i)
    for j in range(i + 1, n): 
        # print("j = ", j)
        # print("my_array[j] = ", my_array[j])
        if my_array[j] < my_array[min_index]:

            min_index = j

    min_value = my_array.pop(min_index)
    print("min_value ", min_value)
    my_array.insert(i,min_value)
    print(my_array)

    

print("Sort array: ", my_array)